components/lib/crash-sentry/src/main/java/mozilla/components/lib/crash/sentry/service/SentryService.kt

```diff
@@ -0,0 +1,167 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.crash.sentry.service
```

Comment:
nit: Extra package nesting is unnecessary.

```suggestion
package mozilla.components.lib.crash.sentry
```